common/maliit/namespace.h

```diff
@@ -16,6 +16,7 @@
 
 #include <QMetaType>
 #include <QSharedPointer>
+#include <QList>
```

Comment:
Why was this needed exactly?

Reply:
Both framework/keyboard still compile fine without it on Qt 6.8.1 for me so seems unneeded.